src/sentry/static/sentry/app/utils/discover/arrayValue.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+
+type Props = {
+  value: string[];
+};
+type State = {
+  expanded: boolean;
+};
+
+class ArrayValue extends React.Component<Props, State> {
+  state: State = {
+    expanded: false,
+  };
+  handleToggle = () => {
+    this.setState(prevState => ({
+      expanded: !prevState.expanded,
+    }));
+  };
+
+  render() {
+    const {expanded} = this.state;
+    const {value} = this.props;
+    return (
+      <ArrayContainer>
+        {expanded &&
+          value
+            .slice(0, value.length - 1)
+            .map(item => <ArrayItem key={item}>{item}</ArrayItem>)}
+        <span>
+          {value.slice(-1)[0]}
+          <button onClick={this.handleToggle}>
+            {expanded ? t('[collapse]') : t('[+%s more]', value.length - 1)}
```

Comment:
When `value` is an empty array this is rendering as `[+-1 more]`

Reply:
Fixed. I added tests for this component as it got past my 'simple' threshold.